crates/bevy_mesh/src/primitives/dim2.rs

```diff
@@ -464,11 +464,11 @@ impl RegularPolygonMeshBuilder {
     ///
     /// Panics if `circumradius` is negative, or if `sides` is less than 3.
```

Comment:
The `Panic` comment is not entirely accurate, as the panics happen only in debug. (At least I think this is how Panic doc comments should be interpreted?)